apps/voting/app/src/components/VoteSummary.js

```diff
@@ -21,7 +21,7 @@ const VoteSummary = ({ vote, tokenSymbol, tokenDecimals, ready }) => {
       <Header>
         <span>
           <Text color={theme.textSecondary} smallcaps>
-            Current support{' '}
+            Votes{' '}
```

Comment:
Similar to the "Min. Quorum" comment above, I don't think "Votes" really captures what is being shown here. Yes, it shows how many votes there have been and which way they have voted. But it also says "(xx% needed)" and `Votes (xx% needed)` doesn't really tell me what I need to know. If you want this to say "Votes" then it should be followed by `(xx% support needed for approval)` or just remove this section in parentheses altogether (in which case users will have to look through the Permissions app to find the support level required on this voting app).

Reply:
Votes (xx% support needed for approval) sounds good to me